similar/main/fireball.cpp

```diff
@@ -1380,25 +1380,31 @@ unsigned do_exploding_wall_frame(wall &w1)
 
 		auto &Walls = LevelUniqueWallSubsystemState.Walls;
 		auto &vmwallptr = Walls.vmptr;
-		auto &w2 = *vmwallptr(csegp->shared_segment::sides[cside].wall_num);
-		assert(&w1 != &w2);
-		assert((w1.flags & WALL_EXPLODING) || (w2.flags & WALL_EXPLODING));
-		w1.flags |= WALL_BLASTED;
-		w2.flags |= WALL_BLASTED;
-		if (w1_explode_time_elapsed >= EXPL_WALL_TIME)
+
+		auto cwall_num = csegp->shared_segment::sides[cside].wall_num;
```

Comment:
I generally make local variables `const` wherever possible, but there is no need to rework this for that.